src/Components/NumberButton.tsx

```diff
@@ -4,20 +4,31 @@ type NumberButtonProps = {
     label: string;
     onIncrement: () => void;
     onDecrement: () => void;
+    horizontal?: boolean;
   };
   
   const NumberButton: React.FC<NumberButtonProps> = ({
     label,
     onIncrement,
     onDecrement,
+    horizontal = false,
   }) => {
-    return (
+    const verticalCounter = (
       <div className='numberCruncher'>
         <button onClick={onIncrement}>▲</button>
         <span>{label}</span>
         <button onClick={onDecrement}>▼</button>
       </div>
     );
+    const horizontalCounter = (
```

Comment:
Here you are setting both consts regardless of the Boolean, it might be neater to pull these out into private components in this file then just render the correct one based on the flag

Reply:
Good point, this is now resolved.